src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -72,7 +72,7 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.8" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.5.1" />
+    <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.5.2-beta.2" />
```

Comment:
Using beta version while validating end to end. Will publish 1.5.2 after all validation succeeds and before merging PR.